index.html

```diff
@@ -10,18 +10,21 @@
 </head>
 
 <body>
+
     <div class="container">
         <div class="col-md-12">
             <h1 id="title">Survey Form</h1>
             <p id="description">Please provide us a short feedback about your experience with Microverse</p>
         </div>
+
```

Comment:
Great Job! 👍 on using the hx tag for the title and using h1 once in your code, it makes your project more semantic.

Reply:
You did well with your review, by using inline comment and making a polite comments, Please it is required that you make at least two inline comments as a way to practice a good review ability 👍